napari/layers/_image_layer/model.py

```diff
@@ -377,8 +377,8 @@ def get_value(self, position, indices):
         """
         transform = self._node.canvas.scene.node_transform(self._node)
         pos = transform.map(position)
-        pos = [clip(pos[1], 0, self.shape[0]-1), clip(pos[0], 0,
-                                                      self.shape[1]-1)]
+        pos = ([clip(pos[1], 0, self._image_view.shape[0]-1),
```

Comment:
why the added parentheses?

Reply:
I thought you needed them to allow for the line continuation! Will remove and then merge - thanks for reviewing the one-line PR :-)